src/lib/models/event-groups/get-group-name.ts

```diff
@@ -19,33 +19,80 @@ export const getEventGroupName = (event: CommonHistoryEvent): string => {
   }
 
   if (isTimerStartedEvent(event)) {
-    return `Timer ${event.timerStartedEventAttributes
+    return `${event.timerStartedEventAttributes
       ?.timerId} (${formatDurationAbbreviated(
       event.timerStartedEventAttributes
         ?.startToFireTimeout as unknown as Duration,
     )})`;
   }
 
   if (isSignalExternalWorkflowExecutionInitiatedEvent(event)) {
-    return `Signal: ${event.signalExternalWorkflowExecutionInitiatedEventAttributes?.signalName}`;
+    return event.signalExternalWorkflowExecutionInitiatedEventAttributes
+      ?.signalName;
   }
 
   if (isWorkflowExecutionSignaledEvent(event)) {
-    return `Signal received: ${event.workflowExecutionSignaledEventAttributes?.signalName}`;
+    return event.workflowExecutionSignaledEventAttributes?.signalName;
   }
 
   if (isMarkerRecordedEvent(event)) {
     if (isLocalActivityMarkerEvent(event)) {
       return 'Local Activity';
     }
-    return `Marker: ${event.markerRecordedEventAttributes?.markerName}`;
+    return event.markerRecordedEventAttributes?.markerName;
   }
 
   if (isStartChildWorkflowExecutionInitiatedEvent(event)) {
-    return `Child Workflow: ${event.startChildWorkflowExecutionInitiatedEventAttributes?.workflowType?.name}`;
+    return event.startChildWorkflowExecutionInitiatedEventAttributes
+      ?.workflowType?.name;
   }
 
   if (isWorkflowExecutionUpdateAcceptedEvent(event)) {
-    return `${event.workflowExecutionUpdateAcceptedEventAttributes?.acceptedRequest?.input?.name}`;
+    return event.workflowExecutionUpdateAcceptedEventAttributes?.acceptedRequest
+      ?.input?.name;
   }
 };
+
+export const getEventGroupLabel = (event: CommonHistoryEvent): string => {
+  if (!event) return;
```

Comment:
Looks like maybe we want to return an empty string here instead of `undefined` based on the type defined for the return?. Also (nit) it looks like this uses `` instead of `''` in a lot of places (maybe something to have auto-fix on format). 

```suggestion
  if (!event) return '';
```

Reply:
👍